### PR TITLE
Improve code coverage

### PR DESCRIPTION
**Pull request by ssbarnea**
Partial-Fix: #27

### CODE REVIEW

File: test/test_runtime.py [hunk 1]
```diff
@@ -71,6 +71,29 @@ def test_runtime_version_fail(mocker: MockerFixture) -> None:
...
+        RuntimeError, match="Unable to find a working copy of ansible executable."
+    ):
+        runtime.version  # pylint: disable=pointless-statement
+
+
+def test_runtime_install_requirements_missing_file() -> None:
+    """Check that missing requirements file is ignored."""
+    # Do not rely on this behavior, it may be removed in the future
+    runtime = Runtime()
+    runtime.install_requirements("/that/does/not/exist")
```
Comment (webknjaz) [inline test/test_runtime.py:94]: Where is the assertion here?

Comment (ssbarnea) [reply test/test_runtime.py:94]: The only thing expected is not to raise, which is implicit. If you want I can assert that it returns None?

Comment (webknjaz) [reply test/test_runtime.py:94]: I think that a more intuitive API would be to actually raise an exception.

Comment (ssbarnea) [reply test/test_runtime.py:94]: I agree with you with only mention that probably we want an optional boolean `raise` option for keeping it like this. Main reason for this is to keep usage simple as the most common pattern is "if reqs file exists install them". A missing file means nothing to install. 

As I am still finding bugs while increasing the coverage, I suspect before making the v1 release and adopting it, we should review all the public API and decide if we like them or not.

**Pull request merged**